### ISSUE TITLE
[BUG] Improve exception handling for incorrect URLs

### ISSUE THREAD

**Issue by RhinosF1** [quirc-bot/QuIRC#62]
pcname:quirc pcname$ python bot.py
loadingconfig
['nick', 'quirctest123', '\n']
['nspassword', 'password', '\n']
['topic', 'channeltopics', '\n']
['greetings', 'hello {}!,Hi {}!,Hello there {}!, Hi there {}!,Hey {}', '\n']
['greetingsbot', '1', '\n']
['weatherbot', '0', '\n']
['owapikey', '000', '\n']
['quotebot', '1', '\n']
['pingbot', '1', '\n']
['buttbot', '1', '\n']
['cashortbot', '1', '\n']
['admins', 'freenode-staff,RhinosF1', '\n']
Starting...
Connected
Sending message...
message sent... logging..
Opened logs
Got timestamp
Logged message

 [NickServ] @ 2019-09-23 22:00:56 - : identify <redacted>
Closed logs
Authed to NickServ
Joined channels
Message recieved
Found channel message
Got message info
opened logs
Got timestamp
Logged message

 [##test1] @ 2019-09-23 22:01:11 - RhinosF1 : .down B|ack0p
Closed logs file
Message recieved
Found channel message
Got message info
opened logs
Got timestamp
Logged message

 [##test1] @ 2019-09-23 22:01:11 - gonzobot : (RhinosF1) http://B|ack0p seems to be down
Closed logs file
Found link
Traceback (most recent call last):
  File "bot.py", line 345, in <module>
    bot.run_loop()
  File "/Users/samuelkirwin/QuIRC/QuIRC.py", line 139, in run_loop
    self.run_once()
  File "/Users/samuelkirwin/QuIRC/QuIRC.py", line 89, in run_once
    event_handler(self, packet.arguments[0], packet.prefix.split("!")[0], packet.arguments[1])
  File "bot.py", line 116, in on_message
    html = requests.get(message_part).text
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/requests/sessions.py", line 519, in request
    prep = self.prepare_request(req)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/requests/sessions.py", line 462, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/requests/models.py", line 313, in prepare
    self.prepare_url(url, params)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/requests/models.py", line 381, in prepare_url
    raise InvalidURL(*e.args)
requests.exceptions.InvalidURL: Failed to parse: http://B|ack0p
Samuels-Air:quirc samuelkirwin$ 

TL;DR:
except InvaildURL:
      continue
